Gui/Button/ButtonBase.cs

```diff
@@ -82,14 +80,7 @@ public override void OnEnabledChanged(EventArgs e)
 
 		override public void OnMouseUp(MouseEventArgs mouseEvent)
 		{
-			if (MouseDownOnButton
-			  && PositionWithinLocalBounds(mouseEvent.X, mouseEvent.Y))
-			{
-				ClickButton(mouseEvent);
-			}
-
 			MouseDownOnButton = false;
```

Comment:
No longer fires click, continues to update local MouseDownOnButton state. Should be moved to and owned by GuiWidget